connectivity/check/action.go

```diff
@@ -119,6 +124,37 @@ func (a *Action) Destination() TestPeer {
 	return a.dst
 }
 
+func (a *Action) startMonitors(opts ...string) {
```

Comment:
could you also call hubble getflows with follow option here, or is it too much to ask...